azure-pipelines.yml

```
@@ -93,6 +93,19 @@ jobs:
           MB_ML_VER: "2014"
           CONFIG_PATH: "config_ilp64.sh"
 
+        py_3.10_32:
+          AZURE_PYTHON_VERSION: "3.9"
+          MB_PYTHON_VERSION: "3.10"
+          PLAT: "i686"
+          MB_ML_VER: "2014"
+          ENV_VARS_PATH: "env_vars_32.sh"
+          DOCKER_TEST_IMAGE: "multibuild/xenial_{PLAT}"
+        py_3.10_64:
+          AZURE_PYTHON_VERSION: "3.9"
+          MB_PYTHON_VERSION: "3.10"
+          MB_ML_VER: "2014"
+          CONFIG_PATH: "config_ilp64.sh"
```

I uploaded focal images, let's give it a try. They will not be available for 32-bit linux
```suggestion
          CONFIG_PATH: "config_ilp64.sh"
          DOCKER_TEST_IMAGE: "multibuild/focal_{PLAT}"
```